ggml/src/ggml-cpu/ggml-cpu-quants.c

```diff
@@ -7132,6 +7752,85 @@ void ggml_vec_dot_q6_K_q8_K(int n, float * restrict s, size_t bs, const void * r
 
     *s = hsum_float_8(acc);
 
+#elif defined __wasm_simd128__
+    int8_t aux8[QK_K] __attribute__((aligned(16)));
+    int32_t aux32[8] __attribute__((aligned(16))) = {0};
+    float sums[8] __attribute__((aligned(16))) = {0};
```

Comment:
Small thing to note here, for this `q6_K_q8_K`, it is very difficult to get the correct result. To make it works, I asked deepseek to invent a new approach without giving it prior examples. That's why the structure of this function is different from the rest.

Reply:
So is it covered with tests?

Reply:
without tests, how can I know that it does not provide a good result in the first place?